Source/CronosPlayUnreal/Public/PlayCppSdkActor.h

```diff
@@ -326,7 +359,124 @@ class CRONOSPLAYUNREAL_API APlayCppSdkActor : public AActor {
+   * @param approvedAddress  address to approve
+   * @param tokenid token id
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   *
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc721Approve", Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc721Approve(FString contractAddress, FString fromAddress,
+                     FString approvedAddress, FString tokenid, FString gasLimit,
+                     FString gasPrice, FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc1155 Moves `amount` tokens from `from_address` to `to_address` using the
+   * allowance mechanism.
+   * @param contractAddress erc1155 contract
+   * @param fromAddress from address to move
+   * @param toAddress to address
+   * @param tokenid token id
+   * @param amount amount
+   * @param additionaldata additional data
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc1155SafeTransferFrom",
+                    Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc1155SafeTransferFrom(FString contractAddress, FString fromAddress,
+                               FString toAddress, FString tokenid,
+                               FString amount, TArray<uint8> additionaldata,
+                               FString gasLimit, FString gasPrice,
+                               FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc1155 Allows `approved_address` to withdraw from your account multiple
+   * times, up to the `amount` amount.
+   * @param contractAddress erc1155 contract
+   * @param fromAddress from address
+   * @param approvedAddress address to approve
+   * @param approved approved or not
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   *
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc1155Approve", Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
+  void Erc1155Approve(FString contractAddress, FString fromAddress,
+                      FString approvedAddress, bool approved, FString gasLimit,
+                      FString gasPrice, FCronosSignedTransactionDelegate Out);
+
+  /**
+   * erc20 Moves `amount` tokens from `from_address` to `to_address` using the
+   * allowance mechanism.
+   * @param contractAddress erc20 contract
+   * @param fromAddress from address to move
+   * @param toAddress to address
+   * @param amount amount
+   * @param gasLimit gas limit
+   * @param gasPrice gas price
+   * @param Out FCronosSignedTransactionDelegate callback
+   */
+  UFUNCTION(BlueprintCallable,
+            meta = (DisplayName = "Erc20TransferFrom", Keywords = "Wallet"),
+            Category = "CronosPlayUnreal")
```

Comment:
Catetory does not align with other WalletConect features, which supposed to be `PlayCppSdk` in PlayCppSdkActor.h